system-tests/_support/index.js

```diff
@@ -159,6 +159,30 @@ Cypress.addChildCommand("contents", function(elements) {
     .get();
 });
 
+/* Sets given JSONString as Value for ACE Editor
+ * 
+ * This function will set the given text as value for ACE Editor
+ *
+ * @param {jQuery.Element} elements - The DOM scope to search within
+ * @param {String} JSONString - JSON Code as String
+ */
+Cypress.addChildCommand("setJSON", function(elements, JSONString) {
+  if (elements.length != null && JSONString) {
+    elements.map(function(index, element) {
+      const doc = element.ownerDocument;
+      const win = doc.defaultView || doc.parentWindow;
+      if (element.classList.contains("ace_editor")) {
+        win.ace.edit(element.id).setValue(JSONString);
```

Comment:
if you're doing other conditional checks above, why not also check that `win.ace` is defined, rather than assuming it will be because an element with class `ace_editor` exists.